_posts/news/2016-04-30-announcing-scalajs-0.6.9.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Announcing Scala.js 0.6.9
+category: news
+tags: [releases]
+permalink: /news/2016/04/30/announcing-scalajs-0.6.9/
+---
+
+
+We are pleased to announce the release of Scala.js 0.6.9!
+
+This release was mostly focused on ironing out the support of Scala 2.12.0-M4 and the upcoming M5.
+All users of 2.12.0-M4 are highly encouraged to upgrade to Scala.js 0.6.9.
+The release also contains a few bug fixes, as well as the following improvements:
+
+* [#2009](https://github.com/scala-js/scala-js/issues/2009) No own class for anonymous Scala.js-defined JS classes (see below)
+* [#2350](https://github.com/scala-js/scala-js/issues/2350) A read-only sbt setting `isScalaJSProject` which is `true` iff the current project is a Scala.js project
+
```

Comment:
Maybe drop a line about the `@JSName` on objects extending `js.GlobalScope`? Technically speaking, this could break code.